DataCollector/DoctrineDataCollector.php

```diff
@@ -222,6 +222,9 @@ public function getGroupedQueries()
             });
             $groupedQueries[$connection] = $connectionGroupedQueries;
         }
+        if ((float) $totalExecutionMS === 0.0) {
```

Comment:
Since the code is there to avoid a division by 0, there's no need for an epsilon check: any non-zero value won't trigger the error condition this check is trying to fix.